NEWS.md

```diff
@@ -0,0 +1,10 @@
+# pacta.aggregate.loanbook.plots (development version)
+
+* Added a `NEWS.md` file to track changes to the package.
+* Replaced references to green/brown with references to increasing/decreasing,
+in line with changes in `r2dii.data` and `r2dii.analysis`
+
+# pacta.aggregate.loanbook.plots 0.0.0.9000
```

Comment:
The current (dev) version on main appears to be 0.0.0.9000, so it's not clear to me why these news items are separated into different sections

Reply:
🤷🏻

Reply:
for now I incremented the package version to 0.0.0.9001 based on https://combine-australia.github.io/r-pkg-dev/versioning.html